vision/unit_tests/test__gax.py

```diff
@@ -32,6 +32,47 @@ def test_ctor(self):
             api = self._make_one(client)
         self.assertIs(api._client, client)
 
+    def test_gapic_credentials(self):
+        from google.cloud.gapic.vision.v1.image_annotator_client import (
+            ImageAnnotatorClient)
+
+        from .test_client import _make_credentials
```

Comment:
I am :-1: on unit test modules being imported. Not so bad to just copy-pasta the `_make_credentials` method?

We could (long-term) have a unit test helpers package that we include?